packages/tools/src/tools/segmentation/BrushTool.ts

```diff
@@ -529,6 +696,17 @@ class BrushTool extends BaseTool {
         color,
       }
     );
+    const circleUID1 = '1';
+    drawCircleSvg(
+      svgDrawingHelper,
+      annotationUID,
+      circleUID1,
+      center as Types.Point2,
+      2,
+      {
+        color,
+      }
+    );
```

Comment:
I think this whole thing should be configurable. Someone might say, "I liked the previous one."